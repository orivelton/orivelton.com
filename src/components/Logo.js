import React from 'react'
import { Link } from "gatsby";

const Logo = () => (
  <Link to="/" className="nav__link" title="Go to Home">⌂</Link>
)

export default Logo;
