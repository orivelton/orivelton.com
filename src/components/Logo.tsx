import Link from 'next/link'
import Icon from '@chakra-ui/icon'
import { GiMoebiusTriangle } from 'react-icons/gi'

const Logo = () => (
  <Link href="/">
    <Icon as={GiMoebiusTriangle} w={45} h={45} />
  </Link>
)

export default Logo;