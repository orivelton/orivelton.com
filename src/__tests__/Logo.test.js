import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../components/Logo';

describe('src/components/Logo.js', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Logo />)
      .toJSON()
    expect(tree).toMatchSnapshot();
  });
});
