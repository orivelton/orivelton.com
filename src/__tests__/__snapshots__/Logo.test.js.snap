// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Logo.js renders correctly 1`] = `
<a
  className="nav__link nav__link--logo"
  href="/"
  title="Go to Home"
>
  <img
    alt="Logo Orivelton Cesar"
    className="logo"
    src="test-file-stub"
  />
</a>
`;
